front/js/components/main.es6

```diff
@@ -38,6 +49,16 @@
     );
   }
 
+  function initSlider() {
+    if ($(window).width() < SCREENS.sm) {
+      $('.tile-about .row').slick({
```

Comment:
We should check what will do the library on window resizing and probably make a handler for transitions between sizes (e.g. `sm -> lg`)

Reply:
i'll subtask it